metaflow/plugins/kfp/kfp.py

```diff
@@ -69,6 +75,12 @@
     ScheduleDecorator,
 )
 
+LOGS_DIR = "/logs"
```

Comment:
Should other location be used? This seems to be working for workflow example project. Maybe `metaflow/logs`?

Reply:
yes, or and ideally in PVC shared folder if a PVC is used.
What is unix convention? maybe `/opt/metaflow_logs` ? and `/opt/metaflow_volume/metaflow_logs` if using a PVC, or just always  `/opt/metaflow_volume/metaflow_logs`

Reply:
Thanks for letting me know about the paths. Updating...